packages/itwin/tree-widget/README.md

```diff
@@ -585,62 +735,122 @@ Where `{tree}` specifies which tree component the feature is of.
 
 ### Example
 
-```ts
+  <!-- [[include: [Presentation.Tree-widget.Telemetry-usage-example-imports, Presentation.Tree-widget.Telemetry-usage-example], tsx]] -->
+  <!-- BEGIN EXTRACTION -->
+
+```tsx
 import { UiItemsManager } from "@itwin/appui-react";
-import { TreeWidgetUiItemsProvider } from "@itwin/tree-widget-react";
-
-UiItemsManager.register(
-  new TreeWidgetUiItemsProvider({
-    onPerformanceMeasured={(feature, elapsedTime) => {
-      telemetryClient.log(`TreeWidget [${feature}] took ${elapsedTime} ms`);
-    }},
-    onFeatureUsed={(feature) => {
-      telemetryClient.log(`TreeWidget [${feature}] used`);
-    }},
-  })
-);
+import { CategoriesTreeComponent, createTreeWidget } from "@itwin/tree-widget-react";
+
+UiItemsManager.register({
+  id: "tree-widget-provider",
+  getWidgets: () => [
+    createTreeWidget({
```

Comment:
I agree - there's no point in having this example now, since `createTreeWidget` requires to provide the tree components. The above sections need an update too - I think simply removing the mentions of `TreeWidgetUiItemsProvider` is enough.

Reply:
Changed